app/models/vm_or_template/operations/relocation.rb

```diff
@@ -60,6 +60,18 @@ def migrate(host, pool = nil, priority = "defaultPriority", state = nil)
     raw_migrate(host, pool, priority, state)
   end
 
+  def migration_status
+    miq_custom_get(:migration_status)
```

Comment:
Using a custom attribute could be provider specific implementation, but should not be universal for all providers. (Otherwise we should formally introduce an attribute in schema).

Reply:
In the base class we don't need to provide a default implementation, which should leave to the subclass.

Reply:
This would mean that we need to check whether method is there on the object. We need to add this check to the code.